syntax = "proto3";

package student.academic.v1;

message RBCompany {
  string id = 1;
  string name = 2;
  string std_name = 3;
  string description = 4;
  Address address = 5;
  MonetaryStock capital_stock = 6;
  repeated CompanyPosition position = 7;
}

message Address {
  string city = 1;
  string zipcode = 2;
  string street = 3;
}

message MonetaryStock {
  float value = 1;
  string currency = 2;
}

message CompanyPosition {
  string person_id = 1;
  string role = 2; 
}
